ddl/ddl_worker.go

```diff
@@ -32,10 +34,53 @@ import (
 // RunWorker indicates if this TiDB server starts DDL worker and can run DDL job.
 var RunWorker = true
 
+type workerType byte
+
+const (
+	normalWorker workerType = 0
+	addIdxWorker workerType = 1
+)
+
+type worker struct {
```

Comment:
What about rename worker to `ddlBackgroundWorker`?

Reply:
This work is in the package of ddl. We can omit the prefix of `ddl`. In DDL old code we have the background worker, so I think the key of `Background` it's not very good.